platforms/ide/problems-api/src/main/java/org/gradle/api/problems/internal/ProblemTransformer.java

```diff
@@ -35,8 +37,6 @@ public interface ProblemTransformer {
      *
      * @param problem the problem to transform
      * @return the transformed problem
-     *
-     * @since 8.5
      */
     Problem transform(Problem problem);
```

Comment:
❕ Do not need to be part of this PR, but maybe -- as the transformer is internal -- this should become `InternalProblem problem`? We eventually always cast to the type.

Reply:
Indeed, and InternalProblemBuilder is missing. I'll try implementing it in this PR and tackle it separately if it becomes too big of a change.